x/pki/x509/x509.go

```diff
@@ -63,8 +63,8 @@ func ToSubjectAsText(subject string) string {
 	oldVIDKey := "1.3.6.1.4.1.37244.2.1"
 	oldPIDKey := "1.3.6.1.4.1.37244.2.2"
 
-	newVIDKey := "vid"
-	newPIDKey := "pid"
+	newVIDKey := "Mvid"
```

Comment:
No, here "vid"/"pid" values need to be preserved, as their goal is to be shown in UI as "vid"/'pid"